firmwares/aeotec/ZW189-A.json

```
@@ -0,0 +1,27 @@
+{
+	"devices": [
+		{
+			"brand": "Aeotec",
+			"model": "ZW189-A",
+			"manufacturerId": "0x0371",
+			"productType": "0x0104",
+			"productId": "0x00bd"
+		}
+	],
+	"upgrades": [
+		{
+			"$if": "firmwareVersion >= 1.0 && firmwareVersion < 1.5",
+			"version": "1.5",
+			"channel": "stable",
+			"region": "usa",
```

We should remove the `region` field from the upgrades, at least for the devices that have a region-specific `productType`. We can leave it in as a comment, or put a comment next to the product type explaining which region that is for.
```suggestion
			// "region": "usa",
```

---

I left a //region comment next to the productId, hopefully that should be ok (but also left the region in the file update array commented out).